From 4ab4eb3a20a122cf91047ada172770097253bf50 Mon Sep 17 00:00:00 2001
Message-Id: <patch-v9-5.9-4ab4eb3a20a-20220203T213350Z-avarab@gmail.com>
In-Reply-To: <cover-v9-0.9-00000000000-20220203T213350Z-avarab@gmail.com>
References: <cover-v6-0.8-00000000000-20211102T122507Z-avarab@gmail.com>
	<cover-v9-0.9-00000000000-20220203T213350Z-avarab@gmail.com>
From: =?UTF-8?q?=C3=86var=20Arnfj=C3=B6r=C3=B0=20Bjarmason?=
 <avarab@gmail.com>
Date: Wed, 2 Feb 2022 14:34:35 +0100
Subject: [PATCH v9 5/9] progress.h: format and be consistent with progress.c
 naming
MIME-Version: 1.0
Content-Type: text/plain; charset=UTF-8
Content-Transfer-Encoding: 8bit

Fix an inconsistency introduced in dc6a0757c4f (make struct progress
an opaque type, 2007-10-30) and rename the "progress" parameters to
stop_progress{,_msg}() to "p_progress". Now these match the
corresponding parameters in the *.c code.

While we're at it let's move the definition of the former below the
latter, a subsequent change will start defining stop_progress() in
terms of stop_progress_msg(). Let's also remove the excess whitespace
at the end of the file.

Signed-off-by: Ævar Arnfjörð Bjarmason <avarab@gmail.com>
---
 progress.h | 5 ++---
 1 file changed, 2 insertions(+), 3 deletions(-)

diff --git a/progress.h b/progress.h
index f1913acf73f..4f6806904a8 100644
--- a/progress.h
+++ b/progress.h
@@ -18,7 +18,6 @@ struct progress *start_sparse_progress(const char *title, uint64_t total);
 struct progress *start_delayed_progress(const char *title, uint64_t total);
 struct progress *start_delayed_sparse_progress(const char *title,
 					       uint64_t total);
-void stop_progress(struct progress **progress);
-void stop_progress_msg(struct progress **progress, const char *msg);
-
+void stop_progress_msg(struct progress **p_progress, const char *msg);
+void stop_progress(struct progress **p_progress);
 #endif
-- 
2.35.1.939.g42bf83caa3d

